server/accounts.go

```diff
@@ -179,7 +179,7 @@ type exportMap struct {
 
 // importMap tracks the imported streams and services.
 type importMap struct {
-	streams  map[string]*streamImport
+	streams  []*streamImport
```

Comment:
Can you look at checkStreamImportsEqual()? This does not look like it was updated but ranges through the map using the key (subject). Not sure how this can even compile that that streams was changed to an array?

Reply:
It can compile of course, but "subj" is now an index in the array, so I don't think that function is doing what it is supposed to now..

Reply:
ok will take a look. I might have missed that one. But would have expected a test or two to fail.

Reply:
ok updated this one, please take a look again if you can.